client/index.html

```diff
@@ -9,6 +9,7 @@
 	<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
 	<meta name="mobile-web-app-capable" content="yes">
 	<meta name="referrer" content="no-referrer">
+	<meta name="theme-color" content="#455164">
```

Comment:
This works well with the default theme, but how does it behave with other themes (this comment applies to colors within the manifest file too)?
My guess is that it's not as nice on other themes, at least for the `theme-color` setting, as the toolbar color would stay blue while the loaded theme is not.

Even when themes include metadata like a global color like this, it will be tricky: this needs to be re-set after the user logs in (assuming themes will be controlled by the user settings)

I'm OK with the background color, but could you double check with the toolbar color and see how it goes?